RRP build error

Getting build error on RP
mainwindow.cpp: In member function 'void MainWindow::setupLogPathAndRedirectStdOut()':
mainwindow.cpp:22360:3: error: 'stdlog' was not declared in this scope
Nakefile:713: recipe for target 'mainwindow.o' failed

I suspect due to changes in commit 4b1b588e69bbc88e657cfa6136a7f20fd02adb3b

--- a/app/gui/qt/mainwindow.h
+++ b/app/gui/qt/mainwindow.h
@@ -279,9 +279,7 @@ private:
     QList<QListWidget *> helpLists;
     QHash<QString, help_entry> helpKeywords;
     std::streambuf *coutbuf;
-#if defined(Q_OS_WIN) || defined(Q_OS_MAC)
     std::ofstream stdlog;
-#endif
 
     SonicPiAPIs *autocomplete;
     QString sample_path, log_path;

